### PR TITLE
Edx bookmark access event caliper implementation

### PR DESCRIPTION
**Pull request by arslanhashmi**
**Trello Link:** [here](https://trello.com/c/azKvNrZH/58-bookmark-event-edxbookmarkaccessed)

**Description:** Edx bookmark access event caliper implementation

**Checks before merge:**

- [ ] Reviewed
- [x] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://edx.devstack.lms:18000/u/verified",
+    "type": "Person"
+  },
+  "eventTime": "2018-12-07T06:33:00.901Z",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Different values for `eventTime`. Please match these.

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "NavigatedTo",
+  "actor": {
+    "id": "http://edx.devstack.lms:18000/u/verified",
+    "type": "Person"
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Shouldn't we add `username` here as well in `name` key?

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "eventTime": "2018-12-07T06:33:00.901Z",
+  "extensions": {
+    "extra_fields": {
+      "accept_language": "en-US,en;q=0.9",
+      "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36",
+      "bookmark_id": "verified,block-v1:ar+CS113x+2018_T3+type@vertical+block@ba4ee5b3905747208702f465275733b6",
+      "component_type": "vertical",
+      "component_usage_id": "block-v1:ar+CS113x+2018_T3+type@vertical+block@ba4ee5b3905747208702f465275733b6",
+      "course_id": "course-v1:ar+CS113x+2018_T3",
+      "event_source": "browser",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Please match `event_source` values.

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  },
+  "id": "urn:uuid:a73e2b78-738f-40e8-a53f-c407ba44c893",
+  "object": {
+    "id": "http://edx.devstack.lms:18000/courses/course-v1:ar+CS113x+2018_T3/bookmarks/",
+    "type": "Page"
+  },
+  "referrer": {
+    "id": "http://edx.devstack.lms:18000/courses/course-v1:ar+CS113x+2018_T3/bookmarks/",
+    "type": "type"
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Please correct the value for `WebPage`.

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+  "eventTime":"2018-12-10T07:00:24.355Z",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Please use same value for `eventTime` as  in current event json.

File: openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  "referrer":{
+    "type":"type",
```
Comment (danialmalik) [inline openedx/features/caliper_tracking/tests/expected/edx.bookmark.accessed.json]: Please correct the value for `type` key.

**Comment by asamolion** [comment]
@arslanhashmi Please respond to @danialmalik's comments and also please rename this branch to match the convention.

**Pull request closed without merging**